packages/good-design/tsconfig.json

```diff
@@ -21,7 +21,10 @@
     "declaration": true,
     "emitDeclarationOnly": false,
     "declarationMap": true,
-    "composite": true
+    "composite": true,
+    "paths": {
+      "react": ["../../node_modules/@types/react"]
```

Comment:
whats that?

Reply:
idk, you added it. it takes react types from root of mono instead of local types
could be this was because of the hoisting limits it had before, and this can now be removed since we do hoisting on local level

Reply:
so if it can be removed, remove it.

Reply:
removed